lib/resource.nunjucks

```diff
@@ -132,26 +132,40 @@
                       <h3>Body</h3>
                       {% for key, b in method.body %}
                         <p><strong>Media Type: {{ key }}</strong></p>
-                        {% if b.formParameters %}
+                        {% if b.formParameters or b.type and not b.schema %}
```

Comment:
Thanks for the reaction. I moved the properties away from the form parameters.
I would like to help on the project but I'm not sure about the best way to display the types so any feedback or ideas are welcome.